docs/src/index.md

```diff
@@ -1,6 +1,55 @@
 # Javis
 
-Javis.jl (Julia mAthematical VISualization) is a visualization and animation package that builds on top of [Luxor.jl](https://github.com/JuliaGraphics/Luxor.jl)
+[![Stable](https://img.shields.io/badge/docs-stable-blue.svg)](https://Wikunia.github.io/Javis.jl/stable)
+[![Dev](https://img.shields.io/badge/docs-dev-blue.svg)](https://Wikunia.github.io/Javis.jl/dev)
+[![Build Status](https://github.com/Wikunia/Javis.jl/workflows/CI/badge.svg)](https://github.com/Wikunia/Javis.jl/actions)
+[![Coverage](https://codecov.io/gh/Wikunia/Javis.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/Wikunia/Javis.jl)
```

Comment:
I would avoid using this file as it's just the readme. Maybe we can give different information on this page. Otherwise we always need to copy and paste the readme over and the user doesn't get any additional information